charts/osm/README.md

```diff
@@ -1,6 +1,6 @@
 # osm
 
-![Version: 0.4.2](https://img.shields.io/badge/Version-0.4.2-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v0.4.2](https://img.shields.io/badge/AppVersion-v0.4.2-informational?style=flat-square)
+![Version: 0.6.0-rc.1](https://img.shields.io/badge/Version-0.6.0--rc.1-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v0.6.0-rc.1](https://img.shields.io/badge/AppVersion-v0.6.0--rc.1-informational?style=flat-square)
```

Comment:
Do we want to be merging rc stuff into main? I'd think it would be easier to directly merge from the v0.6.0 release branch when ready. This is to ensure that the images in main always point to the latest tested release.

Reply:
I think that makes sense. We should update the release guide then to distinguish between pre-releases with regular releases and to skip this step for pre-releases.